vmdk_plugin/utils/plugin_server/plugin_server_linux.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package plugin_server
```

Comment:
From [go build tags](https://golang.org/pkg/go/build/) documentation:
```
Naming a file dns_windows.go will cause it to be included only when building the package for Windows; similarly, math_386.s will be included only when building the package for 32-bit x86.
```
I'm not sure if that answers your question.